lib/logstash/inputs/beats.rb

```diff
@@ -217,7 +217,7 @@ def start_buffer_broker
         while !stop?
           @output_queue << @buffered_queue.take
         end
-      rescue InterruptionException => e
+      rescue java.lang.InterruptedException => e
         # If we are shutting down without waiting the queue to unblock
         # we will get an `InterruptionException` in that context we will not log it.
```

Comment:
Please also update the name of the exception here.

Also, is there a way to test this? I'm guessing that putting here ArgumentError would not cause tests to fail